### PR TITLE
PayPal docs

### PR DESCRIPTION
**Pull request by ivov**
Here are the PayPal docs. The new example workflow is [here](https://n8n.io/workflows/438).

The PayPal app creation process is slightly different now, so I also updated the credentials.

Let me know if you have any comments.

### CODE REVIEW

**Comment by tanay1337** [comment]
Hi @ivov, thanks for the PR! Could you please address the issues mentioned by @smamudhan [here](https://github.com/n8n-io/n8n-docs/pull/13) in this PR as well?

**Comment by ivov** [comment]
Hi @tanay1337! The issues are addressed in the new commit.

In the credentials steps, the casing for `Create app` and `Create App` differs because that's how it's spelled in PayPal.

File: docs/nodes/credentials/PayPal/README.md [hunk 1]
```diff
@@ -8,12 +8,12 @@ Create a [PayPal](https://paypal.com/) account.
...
 ## Using OAuth
 
-1. Access your [PayPal dashboard](https://developer.paypal.com/developer/applications/).
-2. Click on My Apps & Credentials on the left panel.
-3. Create a new app.
-4. Enter app name and create app.
-5. Your client ID and secret will be shown.
-6. Use the client ID and secret with your PayPal node credentials in n8n.
-7. Enter n8n provided redirect URL in configuration. ![Redirect URL Explanation here](../README.md).
+1. Access your [PayPal dashboard](https://developer.paypal.com/developer/applications/)
```
Comment (tanay1337) [inline docs/nodes/credentials/PayPal/README.md:11]: Could you please add a full stop at the end of the statement?

File: docs/nodes/nodes-library/nodes/PayPal/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+### 1. Start node
+
+The Start node exists by default when you create a new workflow.
+
+### 2. PayPal node
+
+1. First of all, you'll have to enter credentials for the PayPal node. You can find out how to do that [here](../../../credentials/PayPal/).
+2. Enter an ID in the *Sender Batch ID* field.
+3. Click on *Execute Node* to run the workflow.
```
Comment (tanay1337) [inline docs/nodes/nodes-library/nodes/PayPal/README.md:40]: Could you please add empty newlines at the end of each markdown file?

**Comment by tanay1337** [comment]
Merged, thanks a lot @ivov 🎉

**Pull request merged**